packages/components/package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@chanzuckerberg/czedi-kit-components",
+  "name": "@chanzuckerberg/eds-components",
   "version": "0.0.1-alpha.7",
-  "description": "React components for czedi-kit",
+  "description": "React components for the education design system",
   "keywords": [
     "design system",
     "components"
   ],
   "author": "Devin Witherspoon <dcwither@gmail.com>",
-  "homepage": "https://github.com/chanzuckerberg/edu-design-system/tree/main/packages/czedi-kit-components#readme",
+  "homepage": "https://github.com/chanzuckerberg/edu-design-system/tree/main/packages/components",
```

Comment:
As far as I can tell, the previous link here wasn't valid. Or maybe was, but is dead now.